### PR TITLE
feat: added footer and input in replay mode

### PR DESCRIPTION
**Pull request by denys-kolomiitsev**
Added footer and input in replay mode.

- Issue #104 - Need to add footer in Replay feature and do not remove message box in case of error or stop generating message

Refs: #104;

### CODE REVIEW

**Comment by denys-kolomiitsev** [comment]
/deploy-review
>Deployment status: success

File: src/components/Chat/Chat.tsx [hunk 1]
```diff
@@ -786,42 +784,39 @@ export const Chat = memo(() => {
...
-                {showReplayControls ? (
-                  <ChatReplayControls
-                    onClickReplayStart={handleReplayStart}
-                    onClickReplayReStart={handleReplayReStart}
-                    showReplayStart={selectedConversations.some(
-                      (conv) => conv.messages.length === 0,
+                {!isPlayback && (
+                  <ChatInput
+                    textareaRef={textareaRef}
+                    isMessagesPresented={selectedConversations.some(
```
Comment (mikitabut) [inline src/components/Chat/Chat.tsx]: Looks like you are doing the same in row 113 - checking for messages length

**Review by mikitabut — DISMISSED** [review]

**Review by mikitabut — APPROVED** [review]

File: src/components/Chat/Chat.tsx (hunk 1 above)
Comment (denys-kolomiitsev) [reply src/components/Chat/Chat.tsx]: Code duplication removed

**Comment by denys-kolomiitsev** [comment]
/deploy-review

>Deployment status: success

**Review by IlyaBondar — APPROVED** [review]

**Pull request merged**